myuw/dao/class_website.py

```diff
@@ -35,14 +35,8 @@ def get_page_title_from_url(url):
         soup = BeautifulSoup(html)
         return soup.title.string
     except DataFailureException as ex:
+        raise
+    except Exception as ex:
```

Comment:
pages without a title will raise an attributeerror.  i think we'd want to avoid logging that